scripts/linters/pylint_extensions.py

```diff
@@ -1062,30 +1062,31 @@ def process_module(self, node):
             node: astroid.scoped_nodes.Function. Node to access module content.
         """
 
-        in_multi_line_comment = False
-        multi_line_indicator = b'"""'
+        is_class_or_function = False
+        is_docstring = False
         file_content = read_from_node(node)
         file_length = len(file_content)
         blank_line_counter = 0
 
         for line_num in python_utils.RANGE(file_length):
             line = file_content[line_num].strip()
+            prev_line = ''
```

Comment:
Should we define this before the for loop?

Reply:
Done.